docs/content/integrations/dbt-cloud.mdx

```diff
@@ -28,78 +44,81 @@ width={1055}
+dbt_cloud_instance = dbt_cloud_resource.configured(
+    {
+        "auth_token": {"env": "DBT_CLOUD_API_TOKEN"},
+        "account_id": {"env": "DBT_CLOUD_ACCOUNT_ID"},
+    }
 )
-
-# create a job that uses your op and resource
-@job(resource_defs={"dbt_cloud": my_dbt_cloud_resource})
-def my_dbt_cloud_job():
-    run_dbt_nightly_sync()
 ```
 
----
+## Step 2: Loading dbt Cloud models as assets
+
+In this step, you'll load the dbt Cloud models managed by a dbt Cloud job into Dagster as assets. For context, a dbt Cloud job defines set of commands to run for a dbt Cloud project. The dbt Cloud models managed by a dbt Cloud job are the models that are run by the job after filtering options are respected.
 
-## Running a dbt Cloud job after another op completes
+Using our dbt Cloud resource, we can retrieve information about the models that the dbt Cloud job is managing.
 
-The `dbt_cloud_run_op` has an optional `start_after` input. If you supply the output of another operation to this input, the dbt Cloud op will not start until that upstream operation successfully completes:
+```python startafter=start_load_assets_from_dbt_cloud_job endbefore=end_load_assets_from_dbt_cloud_job file=/integrations/dbt/dbt_cloud.py dedent=4
+from dagster_dbt import load_assets_from_dbt_cloud_job
 
-```python startafter=start_dbt_cloud_job2 endbefore=end_dbt_cloud_job2 file=/integrations/dbt/dbt_cloud.py dedent=4
-@job(resource_defs={"dbt_cloud": my_dbt_cloud_resource})
-def my_two_op_job():
-    run_dbt_nightly_sync(start_after=another_op())
+# Use the dbt_cloud_instance resource we defined in Step 1, and the job_id from Prerequisites
+dbt_cloud_assets = load_assets_from_dbt_cloud_job(
+    dbt_cloud=dbt_cloud_instance,
+    job_id=33333,
+)
 ```
 
----
+<Note>
+  We assume that your dbt Cloud job only has one command: one of either `dbt
+  run` or `dbt build`, along with any optional command line arguments.
+</Note>
+
+The <PyObject module="dagster_dbt" object="load_assets_from_dbt_cloud_job" /> function loads the dbt Cloud models into Dagster as assets, creating one Dagster asset for each model.
+
+When invoked, the function:
+
+1. Invokes your dbt Cloud job with command overrides to compile your dbt project,
+2. Parses the metadata provided by dbt Cloud, and
+3. Generates a set of software-defined assets reflecting the models in the project managed by the dbt Cloud job. Materializing these assets will run the dbt Cloud job that is represented by the loaded assets.
+
+## Step 3: Schedule dbt Cloud job runs
 
-## Scheduling dbt Cloud jobs
+Now that your dbt Cloud assets, you can define a Dagster job that materializes some or all of these assets, triggering the underlying dbt Cloud job.
 
-You can run dbt Cloud jobs on a schedule in the same way as any other Dagster job:
+Instead of guessing when dbt Cloud should run, you can schedule them based on their upstream or downstream dependencies, external events using a sensor, or cron schedule.
```

Comment:
```suggestion
Instead of guessing when dbt Cloud should run, you can schedule assets based on their upstream or downstream dependencies, external events using a sensor, or a cron schedule.
```